Dockerfile

```diff
@@ -58,6 +58,7 @@ RUN apt-get update && apt-get install -y \
     libunistring2 \
     && rm -rf /var/lib/apt/lists/*
 
+# libsml is linked static => no need to copy
```

Comment:
"statically" (adjektiv vs. adverb), but the code is full of german-isms anyway, so whatever...

Reply:
But if we can fix it, lets do it